.github/workflows/pull_request_merge.yml

```diff
@@ -0,0 +1,207 @@
+name: Pull Request Merge CI
```

Comment:
This name is a bit confusing, it doesn't happen on merge, but rather on pushes to branches (presumably post-merge)
Maybe rename file and action to Post-Merge CI?

Reply:
How about `Post Pull Request Merge CI` ?

Reply:
Sounds ok if a bit too long
Any reason to drop the previous name, simply build.yml?

Reply:
Well it's doing a lot more than just build, running lint, test, bdd test, build, publish charts.